glide.lock

```diff
@@ -1,21 +1,20 @@
-hash: 46cd6d2d3f2840288fad453b3d7d8309d53ed56f84794a6edba7c71968b95142
-updated: 2016-01-29T13:16:26.939463551-08:00
+hash: 7407050cee9bb9ce89e23ef26bce4051cce63d558338a4937f027a18b789e3a1
+updated: 2016-02-11T11:58:44.2533151-08:00
 imports:
 - name: github.com/Azure/go-autorest
-  version: 8ced4882956df3128b6d81a183d8e0757ac825b3
+  version: 6a7d1075d0c4e47463316516c3043b1b91dbdc94
   subpackages:
-  - /autorest
+  - autorest
   - autorest/azure
   - autorest/date
   - autorest/to
 - name: github.com/dgrijalva/jwt-go
-  version: afef698c326bfd906b11659432544e5aae441d44
-  subpackages:
-  - .
+  version: f2193411bd642f7db03249fd79d5292c9b34916a
```

Comment:
I guess we can live with this approach until something breaks. Most of these packages are not very dynamic and less likely to behaviorally break.